pytest_logger/plugin.py

```diff
@@ -13,6 +14,28 @@
 else:
     string_type = str
 
+os_symlink = getattr(os, "symlink", None)
```

Comment:
Can be done like that - then other libraries/plugins wouldn't have os monkey-patched. But if you decide to monkey-patch - I have nothing against.